### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by iveksl2**
@peterhurford @robertzk

### CODE REVIEW

File: documentation/readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  x1 + x2
+}
+```
+
+**Python**: 
+
+Within Python one achieves function documentation through a docstring 
+```python
+def add_two_nums(x1, x2):
+    """Return the sum of 2 integers; int, int -> int"""
```
Comment (peterhurford) [inline documentation/readme.md]: [There are a fair number of Python docstring formats.](http://stackoverflow.com/a/24385103/3317833)

**Comment by iveksl2** [comment]
@peterhurford  good suggestion.  And now?

**Comment by peterhurford** [comment]
LGTM :+1: :shipit:

File: documentation/readme.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+To provide the user human readable documenation on a functions purpose & mapping of input to output.
```
Comment (robertzk) [inline documentation/readme.md:1]: s/documenation/documentation ;)

**Pull request merged**